<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayInputDialog</class>
 <widget class="QDialog" name="DayInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialogt#DayInputDialog {  
    border-image:url(:/icons/background.png) 0 0 0 0 stretch stretch;
    background-position: center;  /* 图片居中 */
    background-repeat: no-repeat; /* 不重复 */
   
    
}</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1401</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#background {  
    border-image:url(:/icons/background.png) 0 0 0 0 stretch stretch;   
    
}</string>
   </property>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>510</y>
      <width>900</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>561</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;</string>
    </property>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="cupsSpinBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>470</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;</string>
    </property>
    <property name="text">
     <string>添加饮品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;</string>
    </property>
    <property name="text">
     <string>删除选中</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>150</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;</string>
    </property>
    <property name="text">
     <string>保存记录</string>
    </property>
   </widget>
   <widget class="QComboBox" name="drinkBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>350</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;</string>
    </property>
    <item>
     <property name="text">
      <string>coffee</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tea</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>milktea</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
